cmd/job-executor-service/main.go

```diff
@@ -223,43 +192,60 @@ func _main(args []string, env envConfig) int {
 
 	log.Printf("Creating new http handler")
 
-	// configure http server to receive cloudevents
-	listeningAddr := fmt.Sprintf("localhost:%d", env.Port)
-	listener, err := net.Listen(
-		"tcp", listeningAddr,
-	)
+	imageFilterList, err := utils.BuildImageAllowList(env.AllowedImageList)
 	if err != nil {
-		log.Fatalf("error listening on tcp %s: %v", listeningAddr, err)
+		log.Fatalf("failed to generate the allowlist, %v", err)
 	}
 
-	p, err := cloudevents.NewHTTP(
-		cloudevents.WithPath(env.Path),
-		cloudevents.WithListener(
-			listener,
-		),
-	)
+	log.Fatal(sdk.NewKeptn(
+		serviceName,
+		sdk.WithTaskHandler(
+			eventWildcard,
```

Comment:
Please add a comment like "handle all events and filter them later in jobEventFilter"